File: UI/EAgroUIApp/src/app/collectioncenter/unverified-collection.ts
export class UnverifiedCollection {
    constructor(
        public collectionId: number,
        public farmerId: number,
        public cropId: number,
        public farmerName: string|undefined,
        public cropName: string|undefined,
        public containerType: string,
        public quantity: number,
        public weight: number,
        public collectionDate: string,
    ) { }
}

File: API/GoodsCollections/Models/UnverifiedCollection.cs

namespace GoodsCollections.Models
{
    public class Collection
    {
        public int CollectionId { get; set; }
        public int FarmerId { get; set; }
        public int CropId { get; set; }
        public string CropName { get; set; } =string.Empty;
        public string ContainerType { get; set; } =string.Empty;
        public int Quantity { get; set; }
        public double Weight { get; set; }
        public DateTime CollectionDate { get; set; }
    }
}